integration/spark2/src/main/scala/org/apache/spark/util/AlterTableUtil.scala

```diff
@@ -114,6 +114,7 @@ object AlterTableUtil {
     val lockLocation = tablePath
     locks.zip(locksAcquired).foreach { case (carbonLock, lockType) =>
       val lockFilePath = lockLocation + CarbonCommonConstants.FILE_SEPARATOR +
```

Comment:
add a static function in `CarbonTablePath` to return the lockFilePath

Reply:
Done